packages/client/src/components/QuestionResult/QuestionResult.styles.ts

```diff
@@ -0,0 +1,50 @@
+import styled, { keyframes } from 'styled-components';
+
+import { Icon } from '@codement/ui/components/Icon/Icon';
+import { theme as t } from '@codement/ui/css/theme';
+import { QuestionResultProps } from './QuestionResult';
+
+const slideUp = keyframes`
+    from {
+        opacity: 0;
+        transform: translate3d(0, 75%, 0);
+    }
+
+    to {
+        opacity: 1;
+        transform: none;
+    }
+`;
+
+export const QuestionResultDiv = styled.div`
+    animation: ${slideUp} 0.9s linear;
+    position: relative;
+    display: flex;
+    width: 71.8rem;
+    height: 8.8rem;
```

Comment:
Remove these because:
1. We want the *context* to determine size
2. These are arbitrary values, and should use a theme value instead

Reply:
I will remove the values. I wasn't sure what to use there and followed the dimensions on the Figma drawing. Just to be clear, when you mean by context, do you mean the text that goes in as children determines the size or the layout of the page/grid determines the size of the component?

Reply:
The latter :). The component that's using it can style it

Reply:
Ah perfect! Makes sense to me.